drivers/sensor/silabs/si7006/si7006.c

```diff
@@ -202,6 +207,10 @@ static int si7006_init(const struct device *dev)
 											\
 	static const struct si7006_config si7006_config_##name##_##inst = {		\
 		.i2c = I2C_DT_SPEC_INST_GET(inst),					\
+		.vin_supply = COND_CODE_1(DT_INST_NODE_HAS_PROP(inst, vin_supply),	\
+					  (DEVICE_DT_GET(DT_PHANDLE(DT_DRV_INST(inst),	\
+								    vin_supply))),	\
+					  (NULL)),					\
```

Comment:
Maybe this can be replaced with  `DEVICE_DT_GET_OR_NULL(DT_PHANDLE(DT_DRV_INST(inst), vin_supply))` ?

Reply:
You got it